generators/rest-api/cleanup.mjs

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable prettier/prettier */
```

Comment:
remove this rabbitMqFIle if not used in rest-api generator.

Reply:
I'll modify that function to remove the rest files for clean-up